server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -125,10 +125,19 @@ export function jsonMorganLoggerMiddleware(
 			};
 			httpMetric.setProperties(properties);
 			const resolveMetric = () => {
-				if (properties.status?.startsWith("2")) {
-					httpMetric.success("Request successful");
+				const retry = req.query.retry;
+				if (retry) {
+					if (properties.status?.startsWith("2")) {
```

Comment:
Duplicating this logic isn't a great idea from a maintainability standpoint IMO. Also, more importantly, burying the "retry" property in the message will make it insanely difficult to query in telemetry. Instead, leave `resolveMetric` alone, and add `retry` to the telemetry properties above.

```ts
const properties = {
  // ...
  [HttpProperties.retryCount]: req.query.retry.,
};
```